doc/manual/cookbooks/prescribed_velocity/prescribed_velocity.cc

```diff
@@ -0,0 +1,223 @@
+#include <deal.II/base/parameter_handler.h>
```

Comment:
I wasn't sure if it's okay to put relative paths in higher level directories in manual.tex (ie ../../cookbooks/prescribed_velocity/...). It definitely makes things simpler.